common/scala/transformEnvironment.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+prefix="CONFIG_"
+configVariables=$(compgen -v | grep $prefix)
+
+props=()
+
+for var in $configVariables
```

Comment:
can you add a comment that documents the contract now between settings in the conf files and their use in the code? maybe not necessarily in this file, but we should write it down somewhere.

Reply:
@markusthoemmes bump ^^